packages/css/src/index.ts

```diff
@@ -16,7 +17,33 @@ export function get(
   return obj === undef ? def : obj
 }
 
-export const defaultBreakpoints = [40, 52, 64].map(n => n + 'em')
+const canUseSymbol = typeof Symbol === 'function' && Symbol.for
+
+const REACT_ELEMENT = canUseSymbol ? Symbol.for('react.element') : 0xeac7
+const FORWARD_REF = canUseSymbol ? Symbol.for('react.forward_ref') : 0xeac7
+const VARIANT_SEPARATOR = ' '
+
+const isMergeableObject = (n) => {
+  return (
+    !!n &&
+    typeof n === 'object' &&
+    n.$$typeof !== REACT_ELEMENT &&
+    n.$$typeof !== FORWARD_REF
+  )
+}
+
+const arrayMerge = (destinationArray, sourceArray, options) => sourceArray
+
+/**
+ * Deeply merge themes
+ */
+export const merge = (a: Theme, b: Theme): Theme =>
+  deepmerge(a, b, { isMergeableObject, arrayMerge })
+
+merge.all = <T = Theme>(...args: Partial<T>[]) =>
+  deepmerge.all<T>(args, { isMergeableObject, arrayMerge })
+
+export const defaultBreakpoints = [40, 52, 64].map((n) => n + 'em')
```

Comment:
Prettier wraps these single-parameter lambdas in parentheses for some reason. I can remove them if necessary.